app/facades/recipe_facade.rb

```diff
@@ -0,0 +1,14 @@
+class RecipeFacade
+  attr_reader :search
+  def initialize(search)
+    @search = search
+  end
+
+  def recipes
+    data = RecipeDatabaseService.new.recipes_by_keyword(@search)
+
+    data[:data].map do |recipe_data|
+      Recipe.new(recipe_data)
+    end
```

Comment:
One consideration here, and it may not be something we run into. If you need to call recipes multiple times within a view, each of those times would then hit the API and map out the data that was already mapped in the previous call of recipes.

_IF_ that becomes the case, it may be worth considering registering `@recipes` as an instance variable on the RecipeFacade, and just set it equal to your existing method. That way when it initializes, it calls the method once, and then that result is just stored in the instance variable rather than calling it each time separately.

Just a small efficiency thing to consider if we ever have to call it multiple times :) Great as is though.